### DISCUSSION TITLE
Audio laggs

### DISCUSSION THREAD

**Discussion by TheShubhendra** [lavalink-devs/Lavalink#576] [Q&A] [answered]
I'm facing lagging issues on songs from SoundCloud.
```
bufferDurationMs: 500 # The duration of the 
NAS buffer. Higher values fare better against longer GC pauses
 frameBufferDurationMs: 2000 # How many milliseconds of audio to keep buffered
```
I found above these 2 lines on the configuration file and I am not if changing these values gonna help me or not.

**Comment by topi314** [comment, marked as answer]
for future reference increasing the `bufferDurationMs` can help with laggy audio but it may not due to slow hardware or bad internet
to diagnose something here more info is needed
